### ISSUE TITLE
YouTube embeds not working properly using Gutenberg

### ISSUE THREAD

**Issue by azzronulation** [alleyinteractive/apple-news#594]
Hey folks,

Since using the Gutenberg editor I’ve come across a problem with embedded videos within the content. Articles containing YouTube videos won’t be published to Apple News and instead receive this error:

`There has been an error with the API: FAILED_TO_PARSE_IMAGE - Failed to parse image with URI [https://www.youtube.com/embed/TMrLH4gr9CE?feature=oembed].`

Any advice on how we can fix this would be awesome!

**Comment by kevinfodness** [comment]
I'm seeing this too. The format of the embeds changed in Gutenberg, so our parser isn't recognizing them properly. We need a patch for this.

**Comment by ducfather** [comment]
Are you still working on this? The last patch was more than 7 months ago.
I get these messages when I try to embed a YouTube video (video block or YouTube block doesn't matter).
```
Error: Unable to process image [https://www.youtube.com/embed/byoEBdVoVpM?feature=oembed]
components -> 7 -> URL

Error: Resource with URI [https://www.youtube.com/embed/byoEBdVoVpM?feature=oembed] not found.
components -> 7
```

Offtopic: Do you have a code documentation to write components for custom gutenberg block or who could I ask :)

**Comment by PeterTUN1** [comment]
If you use the html block and then use the iframe embed from youtube like this it will work:

<iframe width="560" height="315" src="https://www.youtube.com/embed/0PiVzyDqKQs" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

**Comment by tripledm** [comment]
I have multiple instances of random images, anything with Twitter or youtube gives the error.
Sure there must have been a fix by now?

**Comment by KarmaMole** [comment]
Hi! Thanks for all you’re work on this plug-in but just wanted to ask if you guys are working on an update that’ll fix this issue. I’m having this problem on a lot of my posts and i’m dreading having to go through them all and replacing my Gutenberg embeds with the html blocks. I’ll wait if an update is happening, but if not, i’ll have to bite the bullet.

**Comment by kevinfodness** [comment]
Yes, it is part of our v2 release that should be happening in the next few weeks.

**Comment by hartey11** [comment]
@kevinfodness any update on when the v2 release will be?

**Comment by kevinfodness** [comment]
@hartey11 I just released it!

**Comment by kevinfodness** [comment]
This should be fixed as part of v2.0.0, which was just released. I'm closing this issue, but we can open a new one if there are any problems with v2.
